core/src/main/web/app/styles/notebook.scss

```diff
@@ -32,6 +32,10 @@
   }
 }
 
+bk-notebook {
+  position: relative;
```

Comment:
the dropdown isn't positioned to the bk-notebook, it's positioned to the closest positioned parent